src/pages/dmt5/_download-app.tsx

```diff
@@ -115,7 +115,7 @@ const DerivMT5GetApp = () => {
                         <StyledInfoIcon src={InfoIcon} />
                         <InfoText>
                             <Localize
-                                translate_text="_t_For mobile app sign-ups, set the broker code to <0>Deriv Holdings (Guernsey) Limited.<0>_t_"
+                                translate_text="_t_For mobile app sign-ups, enter the company name as <0>Deriv<0>_t_"
```

Comment:
Could you please check this change? we recently changed the MT5 broker name to `Deriv Holdings (Guernsey) Limited` in a high priority card.

Reply:
Thanks @Nuzhy-Deriv..This change was there in the card..okay I am removing it back